Makefile

```diff
@@ -30,6 +30,30 @@ DOCKER_NAME ?= kiali/kiali
 DOCKER_VERSION ?= dev
 DOCKER_TAG = ${DOCKER_NAME}:${DOCKER_VERSION}
 
+# If the IMAGE_PULL_POLICY is not defined and its the short version (eg vX.Y and not vX.Y.Z)
+# then we need to use the image pull policy of 'Always' otherwise the latest update to that
+# branch may not be brought in for users
+ifndef IMAGE_PULL_POLICY
+	SHORT_VERSION=$(shell if [[ ${DOCKER_VERSION} =~ ^v[0-9]+\.[0-9]+$$ ]]; then echo true; fi)
+	ifeq (${SHORT_VERSION}, true)
+		IMAGE_PULL_POLICY = Always
+	endif
+
+	ifeq ("${DOCKER_VERSION}", "latest")
+		IMAGE_PULL_POLICY = Always
+	endif
+
+	ifeq ("${DOCKER_VERSION}", "dev")
```

Comment:
This check we can remove because "IfNotPresent" is the default policy (and that is not a deprecated feature).